addons/spreadsheet/static/src/chart/plugins/chart_odoo_menu_plugin.js

```diff
@@ -31,20 +37,18 @@ export default class ChartOdooMenuPlugin extends CorePlugin {
      * @returns {object | undefined}
      */
     getChartOdooMenu(chartId) {
-        const menuId = this.odooMenuReference[chartId];
-        return menuId ? this.getters.getIrMenu(menuId) : undefined;
-    }
-
-    import(data) {
-        if (data.chartOdooMenusReferences) {
-            this.odooMenuReference = data.chartOdooMenusReferences;
+        try {
+            const definition = this.getters.getChartDefinition(chartId);
+            if (definition.extraData && definition.extraData.odooMenuId) {
```

Comment:
this probably implies a migration from previous versions, right?

Reply:
Yes ... :'(